dev/benchmarks/macrobenchmarks/pubspec.yaml

```diff
@@ -87,8 +99,237 @@ dev_dependencies:
 flutter:
   uses-material-design: true
   assets:
-  - packages/flutter_gallery_assets/food/butternut_squash_soup.png
-  - packages/flutter_gallery_assets/food/cherry_pie.png
-  - assets/999x1000.png
+    - packages/flutter_gallery_assets/food/butternut_squash_soup.png
```

Comment:
Have you found out if there's a way to not have to list all the assets? /cc @jonahwilliams

Reply:
Jonah responded me. Wildcard is not supported for assets, so we have to list them.
I think the best way is to add the new assets to `flutter_gallery_assets`, and bump up the version.

Reply:
I am working on updating `flutter_gallery_assets`.

Reply:
Updated to include all assets.